### PR TITLE
Update Alpine info

### LINKED ISSUE #9575: [Alpine 3.17] Update .NET install page on MS Learn

### LINKED ISSUE #9581: [Alpine 3.21] Update .NET install page on MS Learn

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Updated Alpine info
Checked some of the other distros for updated info

Fixes dotnet/core#9581
Fixes dotnet/core#9575

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/linux-alpine.md](https://github.com/dotnet/docs/blob/266e2358f0a0e5f008ba1f7e5c07fafe1aabd7d8/docs/core/install/linux-alpine.md) | [Install the .NET SDK or the .NET Runtime on Alpine](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-alpine?branch=pr-en-us-43964) |
| [docs/core/install/linux-rhel.md](https://github.com/dotnet/docs/blob/266e2358f0a0e5f008ba1f7e5c07fafe1aabd7d8/docs/core/install/linux-rhel.md) | [Install .NET on RHEL and CentOS Stream](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-rhel?branch=pr-en-us-43964) |
| [docs/core/install/linux-snap-runtime.md](https://github.com/dotnet/docs/blob/266e2358f0a0e5f008ba1f7e5c07fafe1aabd7d8/docs/core/install/linux-snap-runtime.md) | [Install .NET Runtime with Snap](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-snap-runtime?branch=pr-en-us-43964) |
| [docs/core/install/linux-ubuntu-install.md](https://github.com/dotnet/docs/blob/266e2358f0a0e5f008ba1f7e5c07fafe1aabd7d8/docs/core/install/linux-ubuntu-install.md) | [[.NET 9](#tab/dotnet9)](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-install?branch=pr-en-us-43964) |
| [docs/core/install/linux-ubuntu.md](https://github.com/dotnet/docs/blob/266e2358f0a0e5f008ba1f7e5c07fafe1aabd7d8/docs/core/install/linux-ubuntu.md) | [Get OS version info which adds the $ID and $VERSION_ID variables](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu?branch=pr-en-us-43964) |

### CODE REVIEW

File: docs/core/install/includes/linux-install-80-apk.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+To learn how to use the .NET CLI, see [.NET CLI overview](../../tools/index.md).
+
+### Install the runtime
+
+The ASP.NET Core Runtime allows you to run apps that were made with .NET that didn't provide the runtime. The following command installs the ASP.NET Core Runtime, which is the most compatible runtime for .NET. In your terminal, run the following command:
+
+```bash
+sudo apk add aspnetcore8-runtime
```
Comment (BillWagner) [inline docs/core/install/includes/linux-install-80-apk.md:24]: Later in the text, I see that .NET 9 isn't yet supported on Alpine. Is that worth a note here?

File: docs/core/install/includes/linux-install-80-apk.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+As an alternative to the ASP.NET Core Runtime, you can install the .NET Runtime, which doesn't include ASP.NET Core support: replace `aspnetcore8-runtime` in the previous command with `dotnet8-runtime`:
+
+```bash
+sudo apk add dotnet8-runtime
```
Comment (BillWagner) [inline docs/core/install/includes/linux-install-80-apk.md:30]: Same comment on 8 vs. 9

**Review by BillWagner — APPROVED** [review]
This looks great @adegeo 

I had a couple questions, and then you can :shipit:

File: docs/core/install/includes/linux-install-80-apk.md (hunk 1 above)
Comment (adegeo) [reply docs/core/install/includes/linux-install-80-apk.md:24]: Nope! These includes are designed to just list the package manager command per .NET version, regardless of the linux flavor and support. At the moment it just so happens this is the only distro to use this package manager. So they only get included where appropriate.

**Pull request merged**